index.js

```diff
@@ -117,16 +113,15 @@ class BarcodeMask extends React.Component {
     });
   };
 
-  _applyMaskFrameTransparency = () => {
-    let transparency = 0.6;
+  _applyMaskFrameBackgroundColor = () => {
```

Comment:
As this method is now providing more than one style `{ backgroundColor: backgroundColor, flex: 1 }`, let's rename it to `_applyMaskFrameStyle`?

Reply:
Could you also confirm if this default backgroundColor is actually being applied if no backgroundColor prop is passed from outside? `rgba(0, 0, 0, 0.6)`

Apparently, the project I am working with isn't showing the default bkg color, but it shows bkg color if I passed it like `<BarcodeMask backgroundColor='rgba(0,0,0,0.6)'/>`

With transparency it was working by providing default transparency.

Reply:
I have added the default backgroundColor